jdbc/src/main/java/io/kestra/jdbc/repository/AbstractJdbcFlowRepository.java

```diff
@@ -236,30 +279,62 @@ public ArrayListTotal<SearchResult<Flow>> findSourceCode(Pageable pageable, @Nul
             });
     }
 
+    @Override
+    public FlowWithSource create(Flow flow, String flowSource) throws ConstraintViolationException {
+        // control if create is valid
+        taskDefaultService.injectDefaults(flow).validate()
```

Comment:
We can't use this one so lower, it will break the application. 
You need to validate the taskdefault only on the webserver and not on the repositories. 
We assume that the validation is already done here